client/src/charts/NivoCharts.js

```diff
@@ -94,6 +112,37 @@ const fixedSymbolShape = ({
   />
 );
 
+class TableSwitchableGraph extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      show_table: false,
+    };
+  }
+
+  render(){      
+    const {
+      show_table,
+    } = this.state;
+
+    const {
+      graph,
+      table,
+    } = this.props;
+
+    return (
+      <Fragment>
+        { show_table ? table : graph }
+        <TableSwitchButton
+          showing_table={show_table}
+          onClick={() => this.setState({ show_table: !show_table }) }
+        />
```

Comment:
`TableSwitchButton` is plain enough, and specific enough as a sub-component of `TableSwitchableGraph`, that I don't think it needs to be standalone. Might as well in-line it here to reduce the boilerplate and scrolling required in reading this code.

I'd recommend placing the button above the graph/table. Two reasons:
  - height is almost certain to change after it's clicked, so if it's below the graph then it's going to be pushed out from under the mouse. Always a bit disorienting. The text also changes so it might be harder to track it to it's new position. Especially bad if the user immediately wants to switch back to the previous state
  - I believe it's a general a11y practice to place controls above the DOM they modify, or at least clarify (in text or label) their position relative to the content they control. I know this is not likely to be turned on in the text-only version but, for screen reader users who stay on the standard site, this would be a valuable button to do properly.